unicore/distribute/api/validators.py

```diff
@@ -32,3 +35,14 @@ def validate_schema(request):
     else:
         request.schema = schema.to_json()
         request.schema_data = data
+
+
+def repo_url_type_schema_validator(node, value):
+    valid_prefixes = ['git://', 'http://', 'https://', 'ssh://']
```

Comment:
Possibly, but this is for cloning a repository. Regardless we need to sort that problem out and first verify it still is a problem. We're using a few releases later of GitPython now.